mnefun/_utils.py

```diff
@@ -80,6 +80,19 @@ def _handle_dict(entry, subj):
     return out
 
 
+def _handle_decim(decim, sfreq):
+    decim = np.array(decim)
+    assert decim.shape == ()
```

Comment:
I assume you tested and this works, but what's the reasoning for this assertion? Isn't always going to be a 'tuple'..is it like a stealthy `type` check?

Reply:
`np.array(..)` can return arbitrary shapes, this ensures it's a scalar as:
```
>>> np.array(1.).shape
()
>>> np.array([1.]).shape
(1,)
```
etc.